src/views/DissolutionFirm/DissolutionFirm.vue

```diff
@@ -24,6 +23,38 @@
       </v-card>
     </section>
 
+    <!-- Dissolution Dissolution Date -->
+    <section id="document-delivery-section" class="mt-10">
+      <header>
+        <h2>Business Dissolution Date</h2>
+        <p class="mt-4 mb-6">
+          Enter the dissolution date of the business. The dissolution date must be after the business start date and registration date. The dissolution date cannot be in the future.
+        </p>
+      </header>
+      <v-card flat class="mt-6">
+
+       <!-- EDIT SECTION -->
+        <v-row no-gutters>
+          <v-col cols="12" sm="3" class="pr-4">
+            <label class="start-date-title title-label">Dissolution Date</label>
+          </v-col>
+          <v-col cols="12" sm="9" class="pt-4 pt-sm-0" id="start-date-selector">
+            <date-picker
+              id="date-picker"
+              ref="startDateRef"
```

Comment:
Leave the ref out since you're not using it. (It's typically used to call methods in this component, such as validate.)